core/data/init.go

```diff
@@ -0,0 +1,89 @@
+//
+// Copyright (c) 2018
+// Cavium
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package data
+
+import (
+	"strings"
+	"time"
+
+	"github.com/edgexfoundry/edgex-go/core/clients/metadataclients"
+	"github.com/edgexfoundry/edgex-go/core/data/clients"
+	"github.com/edgexfoundry/edgex-go/core/data/messaging"
+	consulclient "github.com/edgexfoundry/edgex-go/support/consul-client"
+	"github.com/edgexfoundry/edgex-go/support/logging-client"
+)
+
+// Global variables
+var dbc clients.DBClient
+var loggingClient logger.LoggingClient
+var ep *messaging.EventPublisher
+var mdc metadataclients.DeviceClient
+var msc metadataclients.ServiceClient
+
+// Heartbeat for the data microservice - send a message to logging service
+func heartbeat() {
+	// Loop forever
+	for true {
```

Comment:
See comment above for Go idiomatic infinite loops: https://tour.golang.org/flowcontrol/4